types/jscodeshift/src/testUtils.d.ts

```diff
@@ -19,3 +19,11 @@ export function defineInlineTest(
   expectedOutputSource: string,
   testName?: string
 ): () => any;
+
+export function runInlineTest(
```

Comment:
Source, for reference: https://github.com/facebook/jscodeshift/blob/2fd5e11f469427d474983b2d1c47be9408677591/src/testUtils.js#L46-L51